po/de.po

```diff
@@ -2375,6 +2375,8 @@ msgid ""
 "Account matcher will decline to match accounts with a different commodity "
 "from the imported split if the import information includes the commodity."
 msgstr ""
+"Der Kontenabgleicher lehnt es ab, Konten mit einer anderen Ware als der "
+"importierten abzugleichen, wenn die Importinformationen die Ware enthalten."
```

Comment:
Commodity (Handelsgut) means currency (Währung) or security (Wertpapier). Suggestions for glossary?